src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.testcases.mobile;
+
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mobile.MobileBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mobile.MobileSignupPageObject;
+import org.testng.annotations.Test;
+
+/**
+ * Created by rcunningham on 6/30/15.
+ */
+public class MobileSignupTests extends NewTestTemplate{
+
+    Credentials credentials = config.getCredentials();
+
+    @Test(groups = {"MobileSignup_001", "MobileSignup", "Mobile"})
+    public void MobileSignup_001_signup() {
```

Comment:
Please don't use MobileSignup_001_ prefixes and "_" in method names. A method should be `succesfulSignup`